complexity/action.yml

```diff
@@ -12,16 +12,46 @@ inputs:
   horrid_threshold:
     description: 'The horrid threshold parameter passed to the invocation of complexity'
     required: false
-    default: 8
+    default: 16
```

Comment:
Why did the complexity score double when the complexity tool has remained the same?

Reply:
So this tool actually takes into account line numbers to determine complexity
Where clang-format splits lines up a lot more (partly due to the 80 character per line limit) than uncrustify was
So that's increased the complexity of some files/functions by a lot
I propose we move to [pmccabe](https://gitlab.com/pmccabe/pmccabe) in the future as it's an open source cyclomatic complexity checker, that returned the same score for the previous versions of these files and the uncrustify version